Fix Yatai service versions in docs

danield137 opened the pull request:
* Link to docker hub images
* Remove specific version from deployment (@parano need to make sure `latest` image always point to the latest release)

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
Current docs can cause an invalid state (installing bento from pip and running old docker image.


## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature and improvements (non-breaking change which adds/improves functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Code Refactoring (internal change which is not user facing)
- [X] Documentation
- [ ] Test, CI, or build
<!--- [ ] Others: describe the type of change if it does not fit to categories listed -->

## Component(s) if applicable
- [ ] BentoService (service definition, dependency management, API input/output adapters)
- [ ] Model Artifact (model serialization, multi-framework support)
- [ ] Model Server (mico-batching, dockerisation, logging, OpenAPI, instruments)
- [ ] YataiService gRPC server (model registry, cloud deployment automation)
- [X] YataiService web server (nodejs HTTP server and web UI)
- [ ] Internal (BentoML's own configuration, logging, utility, exception handling)
- [ ] BentoML CLI

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
<!--- If you plan to update documentation or tests in follow-up, please note -->
- [ ] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

parano commented:
Thank you for fixing this @danield137! 

I've just added the "latest" tag for `bentoml/yatai-service` repo on docker hub, could you change it from `0.8.6` tag to `latest` instead?

I will follow up with a PR to make sure we always tag `latest` when releasing a new docker image.

danield137 commented:
> Thank you for fixing this @danield137!
> 
> I've just added the "latest" tag for `bentoml/yatai-service` repo on docker hub, could you change it from `0.8.6` tag to `latest` instead?
> 
> I will follow up with a PR to make sure we always tag `latest` when releasing a new docker image.

I left the version in the section where it is explained that for each bento version there is a corresponding yatai image. I think It's clearer to leave the version number there. 
There is another section where I removed the version tag, so it falls back to `latest` by default.

I'd suggest to either leave it as is (as knowing that version are correlated is useful) or maybe change the content of the docs a bit.

parano commented:
@danield137 didn't realized docker will fall back to `latest` when there's no tag specified, that all make sense, thanks for explaining! Merging this PR now!
